.github/workflows/osrm-backend.yml

```diff
@@ -88,9 +88,10 @@ jobs:
     - name: Prepare environment
       run: |
         npm ci --ignore-scripts
-        export MASON=${GITHUB_WORKSPACE}/scripts/mason.sh
-        ${MASON} install clang-format 10.0.0
-        echo "$(${MASON} prefix clang-format 10.0.0)/bin" >> $GITHUB_PATH
+        wget https://github.com/muttleyxd/clang-tools-static-binaries/releases/download/master-208096c1/clang-format-10_linux-amd64 -O /usr/local/bin/clang-format-10.0.0
```

Comment:
Extracted it to https://github.com/Project-OSRM/osrm-backend/pull/6391 (it was going to be bigger PR, because I was thinking we would have to update clang-format version)